examples/MagicWeather/app/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.revenuecat.purchases:purchases:4.0.2'
+    implementation 'com.revenuecat.purchases:purchases:5.0.0-rc1'
```

Comment:
do we want to have this as a part of the bump process so we don't forget to update in the future?

Reply:
Agreed! I created a story for it https://app.shortcut.com/revenuecat/story/12669/automatically-increase-version-in-purchases-android-magicweather so we don't forget